### ISSUE TITLE
StackBlur does not work properly when there is an alpha channel in the image.

### ISSUE THREAD

**Issue by flexeo** [flozz/StackBlur#44]
StackBlur does not work properly when there is an alpha channel in the image.

There is a white outline in the result: [[](https://jsfiddle.net/badabou/e3bq8s2r/1](https://jsfiddle.net/badabou/e3bq8s2r/16/)

![stackblur](https://user-images.githubusercontent.com/22362051/155706834-787174b7-062e-4898-9564-792d73d246ec.png)

**Comment by flozz** [comment]
StackBlur is not very good with alpha channels :(

**Comment by flexeo** [comment]
Do you think it's complicated to make a fix?

**Comment by flozz** [comment]
I do not think it is simple to fix, as transparent pixels also have values on R, G and B channels... Here is what your image looks like if you remove the alpha channel:

![Capture d’écran de 2022-03-08 17-37-18](https://user-images.githubusercontent.com/971438/157283588-a664a1bc-02f2-4451-9a21-5c65782df2dd.png)

FYI, I am not the author of this algorithm, I just made little fixes and packaged it on NPM. I am afraid I am not able to make changes like this one... :(
